helpdesk_mgmt_fieldservice_equipment/models/helpdesk_ticket.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2025 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class HelpDeskTicket(models.Model):
+    _inherit = "helpdesk.ticket"
+
+    equipment_id = fields.Many2one(
+        comodel_name="fsm.equipment",
+        string="Equipment",
+        help="Equipment that is used for the helpdesk ticket.",
+        domain="['|', "
+        "('location_id', '=', False), "
+        "('location_id', '=', fsm_location_id)]",
```

Comment:
```suggestion
        "('location_id', '=?', fsm_location_id)]",
```

in case `fsm_location_id` is not set yet, to show all equipments